src/components/style/Flashlight.css

```diff
@@ -1,26 +1,17 @@
 
 #container {
-    margin-left: 100px; /*this is just for an offset*/
+    
     position: relative;
     overflow: hidden;
     filter: brightness(0%);
-    /* max-width: min-content;
-    max-height: min-content;
-    display: inline-block; */
-    background-image: url('../../koala.jpg');
+    background-image: url('../../img/koala.jpg');
     background-repeat: no-repeat;
     background-size: cover;
     width: 50vw;
     height: 30vw;
```

Comment:
You may want to make this larger on mobile? Tested in DevTools and it's a bit small, you can probably give it the width of the container since you're not pushing it against text.